pkg/trace/sampler/coresampler.go

```diff
@@ -24,6 +24,8 @@ const (
 	initialSignatureScoreOffset float64       = 1
 	minSignatureScoreOffset     float64       = 0.01
 	defaultSignatureScoreSlope  float64       = 3
+	// when a sampling rate is above defaultSamplingRateThresholdTo1, it's set at 1
```

Comment:
Nit but can we make sure we respect the Go idiom for commenting identifiers (start with identifier name)?
```go
	// defaultSamplingRateThresholdTo1 defines the maximum allowed sampling rate below 1.
	// If this is surpassed, the rate is set to 1.
```
Or similar.